pkg/image/bootc_disk.go

```diff
@@ -89,5 +91,17 @@ func (img *BootcDiskImage) InstantiateManifestFromContainers(m *manifest.Manifes
 		fmt.Sprintf("%s.vmdk", fileBasename),
 		fmt.Sprintf("%s.vhd", fileBasename),
 	}
+
+	// XXX: copied from https://github.com/osbuild/images/blob/v0.85.0/pkg/image/disk.go#L102
```

Comment:
I have a follupw ready that extracts a common helper but want to add more tests https://github.com/osbuild/images/compare/main...mvo5:raw-tgz2?expand=1to ensure we do not regress GCE images.